### PR TITLE
Changing over to new logo

### LINKED ISSUE #38: As a user, I can see the new logo and favicon along with a blog post
The new logo is currently checked in here. It needs to replace to old logo and old favicon. A blog post also should be produced identifying the logo and favicon is changing and thanking our designer. Keeping the new .svg originals along with the new .png files is good needed. The old favicon and .png files should be deleted. Then the organization of the ['images' folder](https://github.com/pulp/pulpproject.org/tree/gh-pages/images/logo) can probably be consolidated also.

- [x] Update the logo to the new logo
- [x] Update the favicon to the new favicon
- [x] Delete the old logo and favicon files
- [x] Ensure the layout of the ['images' folder](https://github.com/pulp/pulpproject.org/tree/gh-pages/images/logo) is sane.
- [x] Write a blog post announcing and displaying the new logo and favicon.

### PR DESCRIPTION
**Pull request by daviddavis**
Fixes #38

**Note** Will update the date on the blog post before merging.

### CODE REVIEW

**Comment by mhrivnak** [comment]
Let's consider rolling this out with the 2.12 release.

**Comment by daviddavis** [comment]
@mhrivnak cool. works for me.

File: _posts/2017-01-19-new-pulp-logo.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+The old logo, as you may remember, was outdated and it was time for a revamp
+(just in time for Pulp 3.0!). Our new logo was designed by [Maria
+Leonova](http://mashaleonova.wordpress.com) at the Red Hat Brno office.
+
+As part of this change, we're updating both the logo on pulpproject.org as well
+as the favicon for this site. There are two different flavors for the new logo:
+[a small one with just the graphic that represent Pulp](/images/pulp_logo.png)
+and [a larger one that features the word "pulp"](/images/pulp_logo_big.png).
+
+While the old Pulp logo featured very muted colors which may be hard to see for the
```
Comment (bmbouter) [inline _posts/2017-01-19-new-pulp-logo.md]: /the visually impaired/those who see color differently/

File: _posts/2017-01-19-new-pulp-logo.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+---
+title: New Pulp Logo
+author: David Davis
+---
+
+![New Pulp Logo](/images/pulp_logo_big.png)
```
Comment (bmbouter) [inline _posts/2017-01-19-new-pulp-logo.md]: When I rendered this on the site it didn't show correctly for me. It just showed the unclickable text "New Pulp Logo".

**Comment by bmbouter** [comment]
Where did the .svg files go? At the tip of HEAD in pulp/pulpproject.org they are [here](https://github.com/pulp/pulpproject.org/tree/gh-pages/images/logo/originals).

**Review by bmbouter — CHANGES_REQUESTED** [review]
I left some comments and questions.

File: _posts/2017-01-19-new-pulp-logo.md (hunk 2 above)
Comment (daviddavis) [reply _posts/2017-01-19-new-pulp-logo.md]: So I'm using an absolute path which should render properly once we publish this to pulpproject.org. I could use a relative path (`../../../images/pulp_logo_big.png`). It looks kinda ugly though. What do you think?

**Comment by daviddavis** [comment]
@bmbouter sorry my branch was out of date. I removed `/images/logo` and moved the svg files to `/images/svg/logo`. Let me know if there's a better place for them though.

File: _posts/2017-01-19-new-pulp-logo.md (hunk 2 above)
Comment (bmbouter) [reply _posts/2017-01-19-new-pulp-logo.md]: Oh that makes sense. Let's leave as is thanks.

**Review by bmbouter — APPROVED** [review]
Thanks @daviddavis so much for this! I would say merge it when you want including the 2.12 release as @mhrivnak suggests. Note that if we merge this on the same day there will be 2 blog posts in 1 day which is not ideal. This content looks ready so it's up to you two when to merge it.

Note that you'll need to update the date in the post name on the day you merge it to reflect that date. Otherwise it will not show at the top of the blog.

Thanks again for putting this together!

**Comment by daviddavis** [comment]
@mhrivnak when is the release date of 2.12? Do you want to do this maybe like a couple days before or a couple days after?

**Pull request merged**